package.json

```
@@ -7,6 +7,7 @@
     "bench": "vitest bench --no-threads",
     "bench:ci": "CI=true vitest bench --no-threads",
     "build": "tsup",
+    "build:wasm": "cd wasm && wasm-pack build -t nodejs",
```

I think you can just pass wasm in as a path argument instead of cding
```suggestion
    "build:wasm": "wasm-pack build -t nodejs wasm",
```